### DISCUSSION TITLE
Cute - strictly programming related issue!!!

### DISCUSSION THREAD

**Discussion by n4n0b1t3** [smartcontractkit/full-blockchain-solidity-course-py#1042] [Show and tell]
![image](https://user-images.githubusercontent.com/32838961/153756122-1cb41081-e295-4363-a2a2-45edf8df6ba3.png)

**Comment by n4n0b1t3** [comment]
![image](https://user-images.githubusercontent.com/32838961/153756945-cfc90ca1-c9d1-49da-955a-7ccb05e42543.png)

**Comment by cromewar** [comment]
I never thought to find a joke in the repository, hahah this was really good @n4n0b1t3 .

**Reply by n4n0b1t3** [reply]
And a cat!

**Reply by Noveleader** [reply]
Hey how this is joke I didn't get it @cromewar

**Reply by n4n0b1t3** [reply]
It's mimimum instead of minimum

**Reply by Noveleader** [reply]
Too much to notice, sorry about that 😄

**Reply by n4n0b1t3** [reply]
That is in the video, it's in the repository, and participants copy it, it's everywhere :-)

**Reply by Noveleader** [reply]
ohh 😆
